documentation/definition-of-done-components.md

```diff
@@ -0,0 +1,22 @@
+<!-- @license CC0-1.0 -->
+
+# Definition of done - components
+
+## Generic
+
+- You've followed [the contribution guidelines](../README.md#contribution-guidelines)
+- You've added or updated the story of the component
+- Your code has the necessary tests written
+- The component is exported in the relevant `index.ts`
+- You've updated the [changelog](../CHANGELOG.md) unreleased section
+- The component, or all elements of a component for more complex cases, can have an additional class name, to enable restyling
+- The component has an EUPL-1.2 licence and all documentation has a Creative Commons 0 licence (CC0)
+- (willen we hier nog iets zeggen over [progressive enhancement](https://www.gov.uk/service-manual/technology/using-progressive-enhancement)?)
```

Comment:
Weet ik niet. Daar halen we onszelf wel een hoop me op de hals misschien. Laten we in de gaten houden hoe dit in de community wordt gedaan.

Lijsten van principes als deze zijn vaak sterker als ze gestaag groeien – dus laten we nu alleen dingen opnemen waar we nu zeker van zijn en gaandeweg, als we meer besluiten nemen of principes ontdekken, deze opnemen in dit soort lijsten en toelichten.